tsconfig.json

```diff
@@ -6,6 +6,7 @@
       "@common/hooks": ["packages/common/src/hooks"],
       "@common/icons": ["packages/common/src/ui/icons"],
       "@common/intl": ["packages/common/src/intl"],
+      "@common/intl/*": ["packages/common/src/intl/*"],
```

Comment:
This is so i could use `@common/intl/locales` - I'm not 100% sure, but I feel like there's a better way

Reply:
ah. yeah -tricky. not a common use-case.. maybe if the `IntlTestProvider` was under `intl` it wouldn't be an issue 😆 

all good - not sure of a better way to do it!

Reply:
🤣 

...

Sorry but it just made me feel this nice feelings, ya know?